core.go

```diff
@@ -18,6 +18,44 @@ const (
 	zapSentryScopeKey = "_zapsentry_scope_"
 )
 
+var (
+	defaultFrameFilters = FrameFilters{
+		SkipModulePrefixFrameFilter("github.com/TheZeroSlave/zapsentry"),
+		SkipFunctionPrefixFrameFilter("go.uber.org/zap"),
+	}
+)
+
+type (
+	FrameFilter interface {
+		SkipFrame(f sentry.Frame) bool
+	}
+	FrameFilters                  []FrameFilter
+	FrameFilterFunc               func(f sentry.Frame) bool
+	SkipModulePrefixFrameFilter   string
+	SkipFunctionPrefixFrameFilter string
+)
+
+func (f FrameFilterFunc) SkipFrame(frame sentry.Frame) bool {
+	return f(frame)
+}
+
+func (f SkipModulePrefixFrameFilter) SkipFrame(frame sentry.Frame) bool {
+	return strings.HasPrefix(frame.Module, string(f))
+}
+
+func (f SkipFunctionPrefixFrameFilter) SkipFrame(frame sentry.Frame) bool {
+	return strings.HasPrefix(frame.Function, string(f))
+}
+
+func (ff FrameFilters) AnyMatch(frame sentry.Frame) bool {
+	for idx := range ff {
```

Comment:
I'd stick with the well-known `i` variable

Reply:
Not my repo, not my place to say...but `i` and `j` are, I would dare to say, universally understood and standard in most of the languages and projects. I wouldn't mind having `index` variable here if you really wanted to be verbose (I actually prefer this naming style with most other variables), but having a weird hybrid (not short and not verbose) just doesn't make sense to me :D

Reply:
I'll change it to either `index` or `i` can't promise now what it'll be, I tend to `index` to avoid another round of hide and seek :smile: you're right that they are well known and understood but depending on the font personally I've issues to find them